google-calendar/1.0/antora.yml

```diff
@@ -0,0 +1,17 @@
+name: google-calendar-connector
+version: '1.0.0'
+display_version: 1.0 (Mule 4)
+title: Google Calendar Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables access to Google Calendar.
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule-google-calendar-connector
```

Comment:
Double check on the Exchange asset. I have noticed with the new connectors, they are using mule4-google-calendar-connector.